attribute-event-requests/INDI-event.md

```diff
@@ -14,12 +14,13 @@ Guides for using this document can be found in the associated [README.md](README
 | `BASM` | 4.0 | [Bat Mitzvah](#bat-mitzvah) | |
 | `BIRT` | 3.0 | [Birth](#birth) | |
 | `BLES` | 3.0 | Blessing | |
-| `BURI` | 3.0 | [Burial](#burial) | |
+| `BURI` | 3.0 | [Burial](#burial) | one of several types of `BURI` |
 | `CENS` | 3.0 | Census | |
 | `CHR`  | 3.0 | [Christening](#christening) | |
 | `CONF` | 3.0 | Confirmation | |
-| `CREM` | 5.4 | [Cremation](#cremation) | |
+| `CREM` | 5.4 | [Cremation](#cremation) | sometimes encoded as `BURI` prior to 5.4 |
 | `DEAT` | 3.0 | [Death](#death) | |
+| `BURI` | 3.0 | [Depositing remains](#depositing-remains) | |
```

Comment:
Semi-duplicate of line 17.  Move up there in alphabetical order by tag, replacing line 17